modules/webhooks/infrastructure/drizzle/webhook-log-repository-drizzle.ts

```
@@ -1,189 +1,278 @@
+}
+
+function parseDirection(direction: string | null): WebhookDirection | null {
+	if (!direction) return null;
+	if (direction === "inbound" || direction === "outbound") {
+		return direction as WebhookDirection;
+	}
+	return null;
+}
+
+function parseDeliveryStatus(status: string | null): DeliveryStatus | null {
+	if (!status) return null;
```

Biome (2.4.7)

[error] Block statements are preferred in this position.

(lint/style/useBlockStatements)